src/vcpkg/cmakevars.cpp

```diff
@@ -264,7 +264,11 @@ endfunction()
             [&](StringView sv) { lines.emplace_back(sv.begin(), sv.end()); },
             default_working_directory);
 
-        Checks::check_exit(VCPKG_LINE_INFO, exit_code == 0, exit_code == 0 ? "" : Strings::join("\n", lines));
+        Checks::check_exit(VCPKG_LINE_INFO,
```

Comment:
We want to avoid adding new non-localized messages; can you instead do something like:
```cpp
if (exit_code != 0)
{
    msg::print(Color::error, msgCommandFailed /*, parameters */);
    Checks::exit_fail(VCPKG_LINE_INFO);
}
```